query-api/src/main/resources/json-schema/projection/match.json

```diff
@@ -25,7 +25,7 @@
         "match": {
             "$ref": "/json-schema/query/choice.json#/definitions/query"
         },
-        "project": {
+        "projection": {
```

Comment:
Accept both project and projection. Keep "project" section, and add "projection" with same contents